google-cloud-pubsub/lib/google/cloud/pubsub/received_message.rb

```diff
@@ -63,6 +63,49 @@ def ack_id
           @grpc.ack_id
         end
 
+        ##
+        # Returns the delivery attempt counter for the message. The delivery attempt
+        # counter is `1 + (the sum of number of NACKs and number of
+        # ack_deadline exceeds)` for the message.
+        #
+        # A NACK is any call to `ModifyAckDeadline` with a `0` deadline. An `ack_deadline`
+        # exceeds event is whenever a message is not acknowledged within
+        # `ack_deadline`. Note that `ack_deadline` is initially
+        # `Subscription.ackDeadlineSeconds`, but may get extended automatically by
+        # the client library.
+        #
+        # The first delivery of a given message will have this value as `1`. The value
+        # is calculated at best effort and is approximate.
+        #
+        # If a dead letter policy is not set on the subscription, this will be `0`. See
+        # {Topic#subscribe}, {Subscription#dead_letter_topic=} and
+        # {Subscription#dead_letter_max_delivery_attempts=}.
+        #
+        # EXPERIMENTAL: This feature is part of a closed alpha release. This
```

Comment:
If it's part of a closed alpha, please clarify that it is available only to whitelisted partner testers, and note what non-whitelisted users should expect (e.g. `nil`?)

Reply:
@dazuma, @hongalex:

@hongalex says this disclaimer is OK as-is. Should I still update it as @dazuma suggests?

Reply:
@dazuma: Updated.